components/ClipboardCopyButton.js

```diff
@@ -0,0 +1,59 @@
+import { useState } from "react";
```

Comment:
This file shares a lot with `components/ClipboardCopy.js`, can they be combined in some way to remove duplication?